kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -48,11 +48,11 @@ data:
           # See documentation at https://aka.ms/ContainerLogv2 for benefits of v2 schema over v1 schema before opting for "v2" schema
           # containerlog_schema_version = "v2"
        #[log_collection_settings.enable_multiline_logs]
-          # fluent-bit based multiline log collection for .NET, Go, Java, and Python stacktraces. Update stacktrace_languages to specificy which languages to collect stacktraces for(valid inputs: go, java, python, dotnet). Default enables all.
+          # fluent-bit based multiline log collection for .NET, Go, Java, and Python stacktraces. Update stacktrace_languages to specificy which languages to collect stacktraces for(valid inputs: "go", "java", "python", "dotnet"). Default disables all.
```

Comment:
remove this wording "Default disables all." since we want to customer to specify the language applicable for their workloads.